### PR TITLE
Major remove what is update ds in nhse

### PR DESCRIPTION
**Pull request by JRPearson500**
# Description

❓**What**:  Removal of "What is Data Science" as this is not our remit and incomplete which gives a poor initialk impression of a asoirational but unfinished team.   Updated Data Science in NHSE to focus on the three teams.  Included some "how do I learn data science" content in there.  Moves the Articles page to the right as currently underpopulated to be one of the first things users come to.   

🧠**Why?**:  Issue 72 and 104

👨‍💻**How?**:  Content update and changes to .yml

# Checklist:
Have checked for the following:
- [x] The website still builds correctly, and you can view it using `mkdocs serve`.
- [x] There are no new "warnings" from mkdocs
- [x] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [x] Spelling errors
- [x] Consistent capitalization
- [x] Consistent numbers
- [x] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [x] Code snippets don't work
- [x] Images not working
- [x] Links not working

## Where it was tested

- Mac OS Sonoma 14.3

### CODE REVIEW

File: overrides/home.html [hunk 1]
```diff
@@ -320,11 +320,11 @@ <h1>NHS England Data Science</h1>
...
 
         <a
-        href="what_is_data_science/How you can learn Data Science/"
-        title="See our work"
+        href="PhDInterns/"
+        title="PhD Intenrship Scheme"
         class="md-button"
         >
-        How can I learn data science?
+        See our PhD Internship Scheme?
```
Comment (amaiaita) [inline overrides/home.html:327]: does this need a question mark?

Comment (amaiaita) [reply overrides/home.html:327]: also whilst im here, any thoughts on a replacement image for the landing page?

**Review by amaiaita — CHANGES_REQUESTED** [review]

File: docs/about.md [hunk 2]
```diff
@@ -18,58 +20,67 @@ We are the [NHS England](https://www.england.nhs.uk/) Data Science Team.
...
 
--   :material-clock-fast:{ .lg .middle } __Do__
+-   :material-link-variant:{ .lg .middle } __Data Linkage__
 
     ---
 
-    We develop and deploy data science products to make a positive impact on NHS patients and workforce.
+    The Data Linkage Hub aims at providing a unified and quality solution to the data linkage needs in NHS England. Data Science is central to achieving this objective, and it covers many aspects, from the mathematical models of entity resolution and record linkage, to identifying and correcting linkage errors, assessing their impact on downstream applications, and ensuring quality. 
+    
+    See <a href="https://nhsengland.github.io/datascience/our_work/linkage/">Data Linkage Enhancements</a>
```
[the remark this answers is not in the document]
Comment (jrpearson-nhs) [reply docs/about.md]: Updated

File: docs/about.md [hunk 3]
```diff
@@ -9,7 +9,9 @@ hide:
```
[the remark this answers is not in the document]
Comment (jrpearson-nhs) [reply docs/about.md:5]: Done

File: docs/about.md [hunk 4]
```diff
@@ -18,58 +20,67 @@ We are the [NHS England](https://www.england.nhs.uk/) Data Science Team.
 <br/>
 
 [Contact Us (datascience@nhs.net)](mailto:datascience@nhs.net){ .md-button .md-button--primary }
-[See our work](./our_work/index.md){ .md-button}
```
[the remark this answers is not in the document]
Comment (jrpearson-nhs) [reply docs/about.md:21]: Thought it wasn't needed but have put back in

File: overrides/home.html (hunk 1 above)
Comment (jrpearson-nhs) [reply overrides/home.html]: Nope - removed.  

Will think of replacement image for landing page separately

**Review by amaiaita — APPROVED** [review]

**Pull request merged**